run-regression-test.sh

```diff
@@ -141,7 +141,7 @@ if [[ "${RUN}" -ne 1 ]]; then
     exit 0
 fi
 
-if [[ ! -f ${RUN_JAR} ]]; then
+if [ ! -f ${RUN_JAR} ]; then
```

Comment:
Please change to this to pass the `shellcheck`.

```suggestion
if ! test -f ${RUN_JAR:+${RUN_JAR}}; then
```